README.md

```diff
@@ -100,6 +100,9 @@ install mechanism. To run `zebrad`, follow the instructions to compile `zebrad`
 for your platform:
 
 1. Install [`cargo` and `rustc`](https://www.rust-lang.org/tools/install).
+     - You will need version 1.58 or later; if you're using the version shipped with your OS it
+       might be outdated. Run `rustc --version` to check. If it's outdated, install it following
+       the instructions in the link. We always test with the latest `stable` version.
```

Comment:
I don't think we should commit to supporting rust 1.58 unless we have it in CI.

Instead, can we say:
```suggestion
     - Zebra is tested with the latest `stable` Rust version.
        Earlier versions are not supported or tested, but they might work.
        (Rust 1.57 and earlier are not supported, due to missing features.)
```